functions/setSymmetricDifference.js

```diff
@@ -0,0 +1,15 @@
+export default function setSymmetricDifference(array1, array2) {
+  let symmetricArray = []
+  let newArray = array1.concat(array2)
+  for ( let i = 0; i < newArray.length; i++ ) {
+    let temp = newArray.slice()
```

Comment:
slicing with nothing in it doesn't do anything? am i missing something?
![screen shot 2017-01-11 at 5 42 14 pm](https://cloud.githubusercontent.com/assets/17391810/21873710/5b67edd2-d825-11e6-95e3-7b540449ee79.png)